apps/server/src/modules/learnroom/common-cartridge/common-cartridge-enums.ts

```diff
@@ -9,3 +9,10 @@ export enum CommonCartridgeResourceType {
 	WEB_LINK_V1 = 'imswl_xmlv1p1',
 	WEB_LINK_V3 = 'imswl_xmlv1p3',
 }
+
+export enum CommonCartridgeIntendedUseType {
+	ASSIGNMENT = 'assignment',
+	LESSON_PLAN = 'lessonplan',
```

Comment:
maybe lesson_plan as value?

Reply:
This is a value specified in the CC specification. I would also prefare lesson_plan, but the specification says otherwise.